packages/patternlab/styleguide/source/_patterns/03-organisms/by-author/social-links-bar.twig

```diff
@@ -1,4 +1,5 @@
 <section class="ma__social-links-bar">
```

Comment:
@arthurbaghdas Please change the `section` to `div`.

Reply:
Done